deploy/examples/export-external-cluster.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+###############################
+
+USAGE=$(cat <<-'EOF'
+# download the create-external-cluster-resource.py script
+pushd /tmp || exit
+curl -LOs https://raw.githubusercontent.com/rook/rook/release-1.10/deploy/examples/create-external-cluster-resources.py
+popd || exit
+
+# cp script into toolbox pod
+toolbox=$(kubectl -n rook-ceph get pod -l app=rook-ceph-tools -o jsonpath='{.items[*].metadata.name}')
+kubectl -n rook-ceph cp /tmp/create-external-cluster-resources.py $toolbox:/etc/ceph/
+
+# run script via the toolbox pod (you have to specify what you are exporting here, e.g. 'rbd-data-pool-name')
+kubectl -n rook-ceph exec -it $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py \
```

Comment:
If your cluster is not connected, How you can use the toolbox??

You really need to use the external cluster to run the python script.

Reply:
As per the steps ou are defining here I will like you are using the rook toolbox to run it 
kubectl -n rook-ceph exec -it $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py \


Can you share a proper design I am confused what and how actually you are achieving the goal,


What I understood from your parent issue was you need the `yamls` instead f the import script, right?

So for doing that, I would have approached it like,
Maybe created a single manifest file that would be using the env variables,

Reply:
It's a working solution, I used it in a home lab and in a production environment, but there are definitely other ways this goal could be achieved.  Another idea would be to modify the 'import-external-cluster.sh' to produce resource files with an option to apply those with kubectl or not.  What I did was a minimal "hack" to get the resource files by slightly modifying the 'import-external-cluster' and replacing the 'kubectl apply' commands with an output to files.

Reply:
I'm under the impression that you wrote 'import-external-cluster.sh', so I figure you probably have another way in mind to achieve the goal from the parent issue, once you see value in doing so.  I figured this script could be possibly accepted as a working solution in the meantime, and that you'd probably replace it with something better after awhile.  At the least, it shows the kind of work around folks are having to do because there isn't currently an existing easy method to generate the resource files for the gitops use case.

Reply:
Can you please share the full output of this shell run,
And clarify a little on how you are running,

I tried in my local setup but directly running it on cephadm ceph shell it doesn't worked

Reply:
@lknite ^^

Reply:
I responded with a comment to subhamkrai on the [pull request](https://github.com/rook/rook/pull/12712) last week.  On my screen it is displaying right below this conversation.  It shows an example run with status messages to stdout & a folder listing of the resource files produced.  Lastly, it shows the output if someone runs the command without any parameters.  The help displays the syntax, and also shows the user how to generate the .env file.  I think everything is in that comment directed at subhamkrai, rather than copy paste it here, maybe take a look at it and see if it answers these.

Reply:
Last doubt can you please show the output of `env.sh` file.

How the actual output looks like and there descriptions

Reply:
@lknite

Reply:
My .env file looks like this (secrets have been edited):
```
export NAMESPACE=rook-ceph
export ROOK_EXTERNAL_FSID=1ae30c23-8d15-4b24-a953-2f550dccb530
export ROOK_EXTERNAL_USERNAME=client.healthchecker
export ROOK_EXTERNAL_CEPH_MON_DATA=b=10.0.6.21:6789
export ROOK_EXTERNAL_USER_SECRET=AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8A==
export ROOK_EXTERNAL_DASHBOARD_LINK=https://10.0.6.22:8443/
export CSI_RBD_NODE_SECRET=AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA==
export CSI_RBD_NODE_SECRET_NAME=csi-rbd-node
export CSI_RBD_PROVISIONER_SECRET=AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIQ==
export CSI_RBD_PROVISIONER_SECRET_NAME=csi-rbd-provisioner
export CEPHFS_POOL_NAME=cephfs-data0
export CEPHFS_METADATA_POOL_NAME=cephfs-metadata
export CEPHFS_FS_NAME=cephfs
export CSI_CEPHFS_NODE_SECRET=AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJQ==
export CSI_CEPHFS_PROVISIONER_SECRET=AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAyQ==
export CSI_CEPHFS_NODE_SECRET_NAME=csi-cephfs-node
export CSI_CEPHFS_PROVISIONER_SECRET_NAME=csi-cephfs-provisioner
export MONITORING_ENDPOINT=10.0.6.22
export MONITORING_ENDPOINT_PORT=9283
export RBD_POOL_NAME=ceph-rbd
export RGW_POOL_PREFIX=default

```